app/models/concerns/cms/addon/sns_poster.rb

```diff
@@ -0,0 +1,156 @@
+module Cms::Addon
+  module SnsPoster
+    extend ActiveSupport::Concern
+    include Cms::Content
+    extend SS::Addon
+
+   require 'twitter'
+
+    included do
+      field :fbauto,     type: String, default: 'expired'
+      field :twauto,     type: String, default: 'expired'
+      field :deleteauto, type: String, default: 'expired'
+      field :twid, type: String
+      field :twuid, type: String
+      field :fbid, type: String
+      field :fbuid, type: String
+      field :fbpid, type: String
+      permit_params :fbauto, :twauto, :deleteauto, :twid, :twuid, :fbid, :fbuid, :fbpid
+      after_generate_file { post_sns() }
+      after_remove_file { delete_sns() }
```

Comment:
__[rubocop]__
* Do not use parentheses for method calls with no arguments. [:ref](https://github.com/bbatsov/ruby-style-guide#no-args-no-parens)

<!-- Comment by [SideCI](https://sideci.com) -->